### PR TITLE
Drop Enumerable:: namespace for Enumerator

### PR DESCRIPTION
**Pull request by kachick**
In my understanding, We can drop this since dropped X18 supports.

1. Is this pull-request complete?

  - [x] Yes, this pull-request is ready to be reviewed and merged.
  - [ ] No, this pull-request is a work-in-progress.

2. Does this pull request fix an issue with an existing feature or introduce a new feature?

  - [x] It fixes an issue with an existing features.
  - [ ] It introduces a new feature.

3. Does this pull-request include tests?

  - [ ] Yes, it includes tests.
  - [x] No, it does not include tests because the tests already exist.
  - [ ] No, it does not include tests because tests are too difficult to write.
  - [ ] No, it does not include tests because ...

### CODE REVIEW

File: mspec/lib/mspec/helpers.rb [hunk 1]
```diff
@@ -3,7 +3,6 @@
 require 'mspec/helpers/datetime'
 require 'mspec/helpers/ducktype'
 require 'mspec/helpers/encode'
-require 'mspec/helpers/enumerator_class'
```
Comment (kachick) [inline mspec/lib/mspec/helpers.rb]: How we can sync this and https://github.com/rubinius/mspec ? Should I update https://github.com/rubinius/mspec first? 🤔

Comment (brixen) [reply mspec/lib/mspec/helpers.rb]: No one uses rubinius/mspec so you don't need to update it first. At some point, we can copy the changes over, but I don't think it is a big deal.

**Comment by brixen** [comment]
@kachick thanks for working on this! Merge whenever you are happy with it.

File: mspec/lib/mspec/helpers.rb (hunk 1 above)
Comment (kachick) [reply mspec/lib/mspec/helpers.rb]: Thanks for the clarify! 🙇 📝

**Comment by kachick** [comment]
Thank you for the review! 🙏

**Pull request merged**